### DISCUSSION TITLE
[resolvido] Mapos em rede pelo XAMPP

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1551] [Q&A]
Saudações pessoal.
Desculpe se estou usando errado essa área, mas não sei bulhufas de como usar o github.
Como faço para deixar o MAPOS via XAMPP acessível via rede local?
Tentei mudar o apache para iplocal:8080 mas o mapos perde todo laioute. e mesmo assim não acesso de outro local.
![image](https://user-images.githubusercontent.com/68247011/142043317-9080dd0d-a9c9-4156-98e1-3cd96e357659.png)

Eu configurei ele no windows pois sempre tenho erro executar o `composer install --no-dev` no linux.
Então eu instalei o mapos no xampp windwos. copiei a pasta para o thdoc do linux e executei.
No servidor ele funciona legal, mas nunca consegui fazer rodar em rede local.
 Onde estou erando?

**Comment by eclaytonlima** [comment]
Olá, voces tem que digitar o IP do pc que está instalado o mapos na rede
local.
Exemplo: http://192.168.1.150/mapos


Pra mim funciona assim...

**Reply by SuperPorte** [reply]
Mas quando eu faço isso no PC cliente, o IP some e vira localhost.
Ou seja, o mapos deixa o servidor e busca local

**Reply by Pr3d4dor** [reply]
@bernarcom Lá onde fica a configuração da URL do mapos você deve configurar o ip local do computador na rede para que isso não aconteça.

**Comment by SuperPorte** [comment]
Mas no apache ou tem um arquivo para configurar também?Em 17 de nov de 2021 20:05, Gianluca Bine ***@***.***> escreveu:

@bernarcom Lá onde fica a configuração da URL do mapos você deve configurar o ip local do computador na rede para que isso não aconteça.

**Comment by eclaytonlima** [comment]
Em mapos <https://github.com/RamonSilva20/mapos>/application
<https://github.com/RamonSilva20/mapos/tree/master/application>/config
<https://github.com/RamonSilva20/mapos/tree/master/application/config>/
config.php

Na linha 42 altere
$config['base_url'] = 'enter_baseurl';

Para seu IP, exemplo:
$config['base_url'] = 'http://192.168.1.145';

Ou:
$config['base_url'] = 'http://192.168.1.145/mapos';

**Reply by SuperPorte** [reply]
CAAAARAACAAAAAA
Matou a pau!!!!!
Só me falta agora fazer enviar email e mudar o número da OS inicial para adotar 100% de MAPOS aqui no meu MEI!!!!!
Muitíssimo obrigado

**Comment by edmarcos1972** [comment]
Segue sugestão de configuração do arquivo "mapos-4.39.0/application/config"

//$config['base_url'] = 'enter_baseurl'; # padrão que vem no arquivo
$config['base_url'] = 'http://'.$_SERVER['HTTP_HOST'].'/mapos'; # modificado para que possa ser acessado por qualquer url

no meu caso de um servidor linux eu crio um link simbólico /var/www/mapos apontando para a pasta de instalação do mapos

lrwxrwxrwx 1 root root    16 nov 25 00:13 mapos -> ../mapos-4.39.0/

**Comment by pitoche2023** [comment]
estou tendo esse problema porem acontece que se eu colocar o host para outra pasta que não seja o de nome mapos da esse erro nao aparece os layouts, ja mudei o host como ja dito aqui mas nada adianta.
Obs. na versao 4.35.2 funcionar normal, isso acontece na 4.39.0
